### PR TITLE
solution

### PR DESCRIPTION
**Pull request by s0lm1r**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+class Todo {
+	constructor() {
```
Comment (ost-ptk) [inline main.js]: use 2 whitespaces instead of 4

File: main.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+class Todo {
+	constructor() {
+		this.todoList = [];
+		this.id = 1
+	}
+
+	addItem(title, priority) {
```
Comment (ost-ptk) [inline main.js:9]: `... the method returns the unique id assigned to the item (positive integer);`

File: main.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+				priority: priority,
+				id: this.id
+			});
+		this.id++;
+		}
+	}
+
+	removeItem(id) {
+		const currentItem = this.todoList.findIndex(el => el.id === id);
+		if (currentItem === -1) {
```
Comment (gestap) [inline main.js]:
```suggestion
		if (currentItem === -1) return false;
```

**Review by ost-ptk — COMMENTED** [review]
fix whitespaces in document

**Review by ost-ptk — COMMENTED** [review]
you didn't fix all comments

**Comment by s0lm1r** [comment]
[image: image.png]
I thought I fixed them all, maybe did something wrong when tried to commit
this task
[image: image.png]

>
>

**Pull request closed without merging**